Kitodo/src/main/java/org/kitodo/production/helper/CatalogConfigurationImporter.java

```diff
@@ -222,11 +223,14 @@ private List<MappingFile> getMappingFiles(ImportConfiguration configuration) thr
             DAOException, MappingFilesMissingException, URISyntaxException, IOException {
         List<MappingFile> mappingFiles = new LinkedList<>();
         List<MappingFile> allMappingFiles = ServiceManager.getMappingFileService().getAll();
-        if (OPACConfig.getCatalog(configuration.getTitle()).containsKey(MAPPING_FILES)) {
+        try {
+            OPACConfig.getCatalog(configuration.getTitle()).configurationAt(MAPPING_FILES);
             for (String filename : OPACConfig.getXsltMappingFiles(configuration.getTitle())) {
                 mappingFiles.add(getConfiguredMappingFile(allMappingFiles, filename, configuration));
             }
-        } else {
+        } catch (IllegalArgumentException e) {
+            logger.debug("No 'mappingFiles' element found in catalog configuration '" + configuration.getTitle()
```

Comment:
I suggest to change the log level to "info". That would be a more suitable log level cause it is a relative important message out of any debug context.

Reply:
Done